Makefile

```
@@ -110,6 +111,17 @@ test/coverage:
 	@$(BAZEL) coverage //... --combined_report=lcov
 	@echo "$(GREEN)✓ Coverage report generated$(NC)"
 
+## bench: run benchmarks for all packages
+.PHONY: bench
+bench:
```

checkmake (0.2.2)

[warning] Target body for "bench" exceeds allowed length of 5 (7).

(maxbodylength)